projects/robots/robotis/darwin-op/protos/Darwin-op-Hinge2.proto

```diff
@@ -0,0 +1,1839 @@
+#VRML_SIM R2021b utf8
+# license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+# license url: https://cyberbotics.com/webots_assets_license
+# tags: static
+# documentation url: https://www.cyberbotics.com/doc/guide/robotis-op2
+# Robotis' DARwIn-OP is an open platform, miniature humanoid robot used for research, education and outreach.
+# The DARwIn-OP model contains principally:
+# - the 20 DOF
+# - LEDs
+# - accelerometer
+# - gyro
+# - camera
+# - realistic physics
+
+PROTO Darwin-op-Hinge2 [
+  field SFVec3f    translation      0 0 0                      # Is `Transform.translation`.
+  field SFRotation rotation         0 1 0 0                    # Is `Transform.rotation`.
```

Comment:
See https://cyberbotics.com/doc/reference/proto-design-guidelines#translation-and-rotation
```suggestion
  field SFRotation rotation         0 0 1 0                    # Is `Transform.rotation`.
```